python/hail/utils/struct.py

```diff
@@ -15,13 +16,9 @@ class Struct(Mapping, HistoryMixin):
     >>> bar.foo
     >>> bar['foo']
 
-    Note that it is possible to use Hail to define struct fields inside
-    of a key table or variant dataset that do not match python syntax.
-    The name "1kg", for example, will not parse to python because it
-    begins with an integer, which is not an acceptable leading character
-    for an identifier.  There are two ways to access this field:
+    Field names that are not valid Python identifers, like fields with
```

Comment:
Typo in "identifers". I'd prefer:

Field names that are not valid Python identifiers, such as fields that start with numbers or contain spaces, ...